src/components/pages/ProfileList/ProfileListContainer.js

```diff
@@ -9,8 +8,8 @@ import RenderProfileListPage from './RenderProfileListPage';
 
 // Here is an example of using our reusable List component to display some list data to the UI.
 const ProfileList = () => {
-  const { authState } = useOktaAuth();
-
+  //Variable is created as place-holder. Function destroyed while removing Okta
+  const authState = null;
```

Comment:
something is off here possibly w/ spaces/tabs? Not sure but the compiler doesn't like it.